pkg/helpers/helpers.go

```diff
@@ -118,3 +120,15 @@ func AcceleratedNetworkingSupported(sku string) bool {
 	}
 	return true
 }
+
+// GetHomeDir attempts to get the home dir from env
+func GetHomeDir() string {
```

Comment:
A unit test for this new helper function would be great

Reply:
Sure, I'm just not sure how to adequately test it without basically putting the same exact logic in the test itself.